docs/guides/index.mdx

```diff
@@ -55,6 +70,11 @@ Creating utility-scale quantum applications generally requires a variety of comp
 - Persist long-running workloads in the cloud, even if your laptop is turned off
 - Deploy reusable programs in the cloud
 
+If you'd like to start using Qiskit Serverless right, install it with pip:
```

Comment:
```suggestion
To start using Qiskit Serverless right away, install it with pip:
```

Reply:
Is this what you meant @Eric-Arellano?

Reply:
Yes, Abby, that is what I meant. I'm also not sure we need to say "right away"?

I also meant to say that we should probably use consistent wording with the prefix sentence for all the install snippets, such as `The catalog can be installed via pip:`

Reply:
Here's what IBM Style says:

> Use this term, which means by way of, to refer to routing data, such as in “Messages are routed via SMTP”. In
other situations, try to replace via with one of the following synonyms: across, along, by, from, on, or through.

"Routing data" seems very specific - do we consider installing a package the same as data being routed to your machine?

Reply:
Sort of? I think Becky normally uses "with pip"? That should translate better.